packages/neuron-ui/src/widgets/Toast/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { ReactNode, useEffect, useState } from 'react'
+import { ReactComponent as SuccessCircle } from 'widgets/Icons/SuccessCircle.svg'
+
+import styles from './toast.module.scss'
+
+interface NotificationProps {
+  type?: 'success' | 'error' | 'info'
+  content: string | ReactNode
+  onDismiss?: () => void
+}
+
+const Toast = ({ type = 'success', content, onDismiss }: NotificationProps) => {
+  const [show, setShow] = useState(false)
+
+  useEffect(() => {
+    if (content) {
```

Comment:
I'm not sure whether we need to remove the timer or not before we set a new one.